src/sentry/runner/commands/tsdb.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import
+
+from collections import OrderedDict
+from datetime import datetime, timedelta
+
+import click
+import pytz
+from dateutil.parser import parse
+
+from sentry.runner.decorators import configuration
+from sentry.utils.iterators import chunked
+
+
+class DateTimeParamType(click.ParamType):
+    name = 'datetime'
+
+    def convert(self, context, option, value):
+        if value is None:
+            return value
+        elif isinstance(value, datetime):
+            return value
+
+        try:
+            result = parse(value)
+        except Exception:
+            self.fail(
+                '{!r} is not a valid datetime'.format(value),
+                option,
+                context,
+            )
+
+        if result.tzinfo is None:
+            # TODO: We should probably warn about this? Also note that this
+            # doesn't use the Django specified timezone, since settings haven't
+            # been configured yet.
+            result = result.replace(tzinfo=pytz.utc)
+
+        return result
+
+
+@click.group()
+def tsdb():
+    """Tools for interacting with the time series database."""
+    pass
+
+
+@tsdb.group()
+def query():
+    """Execute queries against the time series database."""
+    pass
+
+
+@query.command()
+@click.argument(
+    'metrics',
+    nargs=-1,
+    type=click.Choice([
```

Comment:
Oh, nevermind, I see what you mean. You can't import the module at all without everything being configured.